### PR TITLE
Add command for new parser nvme_core_io_timeout

### PR DESCRIPTION
**Pull request by lonicerae**

### CODE REVIEW

**Comment by xiangce** [comment]
Looks good to me

File: uploader.json [hunk 1]
```diff
@@ -2810,6 +2810,11 @@
                 "reserved_port/range"
             ],
             "symbolic_name": "sap_host_profile"
+        },
+        {
+            "file": "/sys/module/nvme_core/parameters/io_timeout",
+            "pattern": [],
+            "symbolic_name": "nvme_core_io_timeout"
```
Comment (bfahr) [inline uploader.json]: @lonicerae please move this spec up so that the symbolic name is in sorted, alphabet order.

File: uploader.v2.json [hunk 2]
```diff
@@ -2805,6 +2805,11 @@
                 "reserved_port/range"
             ],
             "symbolic_name": "sap_host_profile"
+        },
+        {
+            "file": "/sys/module/nvme_core/parameters/io_timeout",
+            "pattern": [],
+            "symbolic_name": "nvme_core_io_timeout"
```
Comment (bfahr) [inline uploader.v2.json]: Same comment as above.

File: uploader.json (hunk 1 above)
Comment (lonicerae) [reply uploader.json]: Okay, got it.

File: uploader.v2.json (hunk 2 above)
Comment (lonicerae) [reply uploader.v2.json]: Got it.

**Comment by lonicerae** [comment]
Hey @bfahr , please help to review it again, thanks!

**Pull request merged**